### ISSUE TITLE
`import/extensions` incorrectly fails an import with extra dots in filename

### ISSUE THREAD

**Issue by paulnelson2** [import-js/eslint-plugin-import#2394]
Given a file:

`group.subgroup.js`

and a rule:

```
"import/extensions": ["error", "never"]
```

and an import statement:

```
import "./group.subgroup";
```

It's incorrectly flagged as an error, despite not using the extension on that import.

I also tried this:

```
"import/extensions": [
      "error",
      {
        js: "never",
      },
    ]
```

Versions:
eslint v7.32.0
eslint-plugin-import v2.25.4

**Comment by paulnelson2** [comment]
Oops, this actually doesn't happen with a fresh reproducible project. I guess it's related to something specific to my project. Closing.
